.trunk/configs/.markdownlint.yaml

```diff
@@ -9,3 +9,4 @@ spaces: false
 url: false
 whitespace: false
 MD041: false
+MD046: false
```

Comment:
This is really nice that you figure out all of this! I was struggling trying to figure it out